logic.py

```
@@ -731,3 +739,52 @@ def _is_candidate_valid(col):
+        # Build OR results from fitted model
+        or_results = {}
+        for col in covariate_cols:
+            if col in result.params.index:
+                or_results[col] = {
+                    'or': np.exp(result.params[col]),
+                    'ci_low': np.exp(result.conf_int().loc[col, 0]),
+                    'ci_high': np.exp(result.conf_int().loc[col, 1]),
+                    'p_value': result.pvalues[col]
+                }
+        
+        return None, or_results, "OK", metrics  # html_table optional
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)